static/css/components/cbox.less

```diff
@@ -22,9 +22,7 @@
 #cbox {
   &Overlay {
     position: fixed;
-    width: 100%;
-    height: 100%;
-    background: @brown-3a301e;
+    background: @grey; //change color here
```

Comment:
You don't need a comment here. This comment will look very strange to someone reading this file 3 years from now :P Ditto for the other ones.